src/gateway/tasks/loadPeers.ts

```diff
@@ -66,6 +71,8 @@ export async function loadPeers(context: Application.BaseContext) {
   for (const peer of newPeers) {
     logger.debug(`Checking Arweave peer ${peer} [${(newPeers.indexOf(peer) + 1)} / ${newPeers.length}]`);
     try {
+      // For me it's a bit misleading that you use Benchmark module from the redstone-smartweave module
+      // Maybe we can create a separate NPM module for benchmarking or check if there are ready solutions
```

Comment:
yup, that's the plan for the nearest future - to move both the logger and  benchmark to a separate utility lib...